FirebaseAI/CHANGELOG.md

```diff
@@ -1,197 +1,6 @@
-# Unreleased
-- [fixed] Fixed `ModalityTokenCount` decoding when the `tokenCount` field is
-  omitted; this occurs when the count is 0. (#14745)
-
-# 11.12.0
```

Comment:
Consider keeping these here with an additional note, since we'll probably delete the other directory at 12.0.0

Reply:
Sounds reasonable to me. Any suggestions for the note? And do you know if this will cause any issues with @morganchen12's release note generation?